docs/release-notes/NuGet-5.11.md

```diff
@@ -27,6 +27,9 @@ NuGet distribution vehicles:
 
 * [Security]: Microsoft Security Advisory CVE-2023-29337 | NuGet Client Remote Code Execution Vulnerability - [#12653](https://github.com/NuGet/Home/issues/12653)
 
+> [!NOTE]
+> There is a behavior breaking change on Linux. The temp folder location, where NuGet stores temporary files during its various operations, has changed from `/tmp/NuGetScratch` to `/tmp/NuGetScratch<username>` on Linux. E.g. for user User1, the temp folder will be `/tmp/NuGetScratchUser1`.
```

Comment:
I feel bad, but I just noticed now that we say `On Linux`  at the beginning and at the end. 

```suggestion
> There is a behavior breaking change on Linux. The temp folder location, where NuGet stores temporary files during its various operations, has changed from `/tmp/NuGetScratch` to `/tmp/NuGetScratch<username>`. E.g. for user User1, the temp folder will be `/tmp/NuGetScratchUser1`.
```

It's not awful so you can choose to ignore/address it.

I *think* the approvals are not dismissed on the docs repo though, so you should be  able to just change it.

Reply:
No worries :) Thanks for reviewing it again! Fixed accordingly~